### ISSUE TITLE
Creating queryParams from ArrayList does not work properly

### ISSUE THREAD

**Issue by kuceraf** [oanda/v20-java-samples#6]
Hi,
I'm trying to get prices as in you example class (PricePolling.java):
```
List<String> instrumentList = Arrays.asList("EUR_USD", "USD_JPY", "GBP_USD", "USD_CHF");
PricingGetRequest request = new PricingGetRequest(new AccountID(accountId), instrumentList);
oandaContext.pricing.get(request);
```
But the Java wrapper library com.oanda.v20:3.0.18 class com.oanda.v20.Context converts the instrumentList to http request call with queryParams looking like this:
**https://api-fxpractice.oanda.com/v3/accounts/101-004-6646999-001/pricing?instruments=%5BEUR_USD%2C+USD_JPY%2C+GBP_USD%2C+USD_CHF%5D**
which results in HTTP 400 error.

http request call must be converted by the library to this request:
**https://api-fxpractice.oanda.com/v3/accounts/101-004-6646999-001/pricing?instruments=EUR_USD%2CUSD_JPY%2CGBP_USD%2CUSD_CHF**

I thing the error happend at class com.oanda.v20.Context, line 120 ( uriBuilder.addParameter(key, value.toString());)

**Comment by kuceraf** [comment]
Are you going to fix it or should I avoid using the Java api wrapper?

**Comment by HerbSears-OANDA** [comment]
This problem was fixed in the 3.0.19 release of the v20 API - please update to that version and retry.
